### DISCUSSION TITLE
Ground markers on minimap after new minimap zoom update are blurry

### DISCUSSION THREAD

**Discussion by Aertig** [runelite/runelite#15776] [Q&A] [answered]
As per title, it seems like tiles marked with the Ground Markers plugin and "Draw tiles on minimap" enabled are now showing as very blurry and seem to be larger than the single marked square. I don't have an image of before, so attached is an image I was able to find from googling, when the functionality was originally released, and an image from today.

Edit: This is with minimap zoom disabled, I understand that zooming/scaling would cause this sort of blur

![D331JC8X4AAVd4c](https://user-images.githubusercontent.com/74419052/194408894-58fbfdb5-a99a-4522-b5e3-6bb4ccb7ecee.jpg)

![image](https://user-images.githubusercontent.com/74419052/194409071-505caeae-3144-4445-a3f7-a351bea8ce67.png)

**Comment by Enriath** [comment, marked as answer]
This has been fixed internally afaik and will be in the next update.

**Reply by Aertig** [reply]
Neat thank you very much, curious if it is fixed for just no zoom change, or both?
